src/app/organizations/collections/state/create-collection.service.ts

```diff
@@ -141,14 +143,16 @@ export class CreateCollectionService {
    *
    * @param {FormsState['createOrUpdateCollection']} collectionFormState
    * @param {number} collectionID  ID of the collection to update
+   * @param {string} collectionDescripton
    * @memberof CreateCollectionService
    */
-  updateCollection(collectionFormState: FormsState['createOrUpdateCollection'], collectionID: number) {
+  updateCollection(collectionFormState: FormsState['createOrUpdateCollection'], collectionID: number, collectionDescripton: string) {
     let collection: Collection;
     collection = {
       name: collectionFormState.name,
       topic: collectionFormState.topic,
-      displayName: collectionFormState.displayName
+      displayName: collectionFormState.displayName,
```

Comment:
Code comment:
Why are the other three coming from state but description is passed in?